stdlib/importlib/machinery.pyi

```diff
@@ -158,3 +159,18 @@ class ExtensionFileLoader(importlib.abc.ExecutionLoader):
     def get_code(self, fullname: str) -> None: ...
     def __eq__(self, other: object) -> bool: ...
     def __hash__(self) -> int: ...
+
+if sys.version_info >= (3, 11):
+    import importlib.readers
+
+    class NamespaceLoader(importlib.abc.InspectLoader):
```

Comment:
> It might also make sense to include `create_module()`, `exec_module()`, and `load_module()` here. While they are inherited in the stubs, they aren't in the implementation (since `NamespaceLoader` inherits directly from `object`).

Good shout. Done!